tests/fp8/test_fp8_parameter.py

```diff
@@ -0,0 +1,25 @@
+import torch
+from nanotron.fp8.constants import DTypes
+from nanotron.fp8.meta import FP8Meta
+from nanotron.fp8.parameter import FP8Parameter
+from nanotron.fp8.tensor import FP8Tensor
+
+
+# @pytest.mark.parametrize("quantize_input", [True, False])
+def test_create_fp8_parameter():
+    tensor = torch.randn(16, 16, device="cuda", dtype=torch.float32)
+    # tensor = FP8Tensor(tensor, DTypes.FP8E4M3) if quantize_input else tensor
+
+    fp8_parameter = FP8Parameter(tensor, DTypes.FP8E4M3)
+
+    # TODO(xrsrke): are there better ways for testing
+    # if a tensor is quantized or not?
```

Comment:
Depending on how you quantize it, you can test the correctness of the quantized value to be exacly what we want